tests/serverpod_test_server/test_integration/auth_core/auth_header_test.dart

```diff
@@ -73,5 +73,61 @@ void main() async {
       var receivedToken = await tokenInspectionCompleter.future;
       expect(receivedToken, key);
     });
+
+    test(
+        'then old clients passing keys in body should still work with new server code',
+        () async {
+      var key = 'username-4713:password-4713';
+
+      // Intentionally clearing the authKeyManager so that auth is not passed in header.
+      await authKeyManager.remove();
+
```

Comment:
Suggestion: Add this as a `tearDown` instead.

Having a `tearDown` in the test implementation is a strong signal that other tests are leaky. Please ensure that this is not required or place it as a common tearDown.